packages/superset-ui-legacy-preset-chart-deckgl/package.json

```diff
@@ -27,6 +27,8 @@
   },
   "dependencies": {
     "@math.gl/web-mercator": "^3.1.3",
+    "@superset-ui/chart-controls": "^0.14.9",
```

Comment:
`peerDependencies`

Reply:
If I add to the peerdependencies  the tests fails. Any idea how to fix this?

Reply:
when you add a lib to `peerDependencies`, and want to test, also add it to `devDependencies`.